modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -3,15 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Partner Manager enables you to manage a large number of trading partners and their message flows. A message flow is the path along which a message travels, from its source, through a transformation (via a DataWeave map), and to its target.
+Anypoint Partner Manager enables you to manage a large number of trading partners and their message flows.
+A message flow is the path along which a message travels from its source, through a DataWeave transformation map, to its target.
```

Comment:
```suggestion
A transmission is a single communication event during which a payload is transferred between you and one of your partners. These payloads can contain one or more messages. The message flows you create determine where the transmissions are received or sent and the path along which a message travels from its source, through a DataWeave transformation map, to its target. 
```

Reply:
You suggested def has a very long sentence at  the end. I can try to condense it, but we could end up in a long debate about it. Plus, it's different from what Amit has sanctioned. I've ask you both to come to an agreement on the definition.

Reply:
Will use Amit's def per David.